Index: gpdb/contrib/auto_explain/auto_explain.c
===================================================================
--- gpdb.orig/contrib/auto_explain/auto_explain.c
+++ gpdb/contrib/auto_explain/auto_explain.c
@@ -342,7 +342,7 @@ explain_ExecutorEnd(QueryDesc *queryDesc
 			es.format = auto_explain_log_format;
 
 			ExplainBeginOutput(&es);
-			ExplainQueryText(&es, queryDesc);
+			//ExplainQueryText(&es, queryDesc);
 			ExplainPrintPlan(&es, queryDesc);
 			if (es.analyze && auto_explain_log_triggers)
 				ExplainPrintTriggers(&es, queryDesc);
Index: gpdb/src/include/pg_config_manual.h
===================================================================
--- gpdb.orig/src/include/pg_config_manual.h
+++ gpdb/src/include/pg_config_manual.h
@@ -20,7 +20,7 @@
  *
  * Changing this requires an initdb.
  */
-#define NAMEDATALEN 64
+#define NAMEDATALEN 128
 
 /*
  * Maximum number of arguments to a function.
Index: gpdb/src/interfaces/ecpg/include/sqlda-native.h
===================================================================
--- gpdb.orig/src/interfaces/ecpg/include/sqlda-native.h
+++ gpdb/src/interfaces/ecpg/include/sqlda-native.h
@@ -13,7 +13,7 @@
  * This should be at least as much as NAMEDATALEN of the database the
  * applications run against.
  */
-#define NAMEDATALEN 64
+#define NAMEDATALEN 128
 
 struct sqlname
 {
